### ISSUE TITLE
Sort-Field missing in typo3-backend of a contact-entry in Personnel

### ISSUE THREAD

**Issue by kal60** [t3brightside/personnel#16]
Where can I find the sorting-Field in the typo3 backend. Has it to be configured to be shown ?
Regards
Knut
Personnel V4.2.0

**Comment by t3brightside** [comment]
Hey @kal60, should be there indeed for the "from pages" content type. For "selected" it's not available.

**Comment by t3brightside** [comment]
@kal60 indeed, as I said it's not available for single records content type. It's by design. If you think it's missing somewhere it supposed to be present as feature, please add full screenshot from debug mode to your explanation, otherwise it's a little hard to make sure we speaking about the same thing.

**Comment by kal60** [comment]
ok, another way to describe: how do I define a sorting-key ? right now it seems to be set automatically when I create a new contact-record ?
(I'll gather some screenshots, will take a while because I've to create some test-records without real person-data)
